frontends/main/src/app-pages/CertificatePage/DigitalCredentialDialog.tsx

```diff
@@ -0,0 +1,212 @@
+    "0 3px 8px 0 rgba(37, 38, 43, 0.12), 0 2px 4px 0 rgba(37, 38, 43, 0.10)",
+}))
+
+const Info = styled.dl({
+  display: "flex",
+  flexDirection: "row",
+  gap: "16px",
+  alignItems: "stretch",
+  marginBottom: "28px",
+})
+
+const InfoColumn = styled.dl(({ theme }) => ({
+  flexGrow: 2,
+  alignSelf: "stretch",
+  ":last-child": {
+    borderLeft: `1px solid ${theme.custom.colors.lightGray2}`,
+    paddingLeft: "16px",
+    flexGrow: 1,
+  },
+}))
+
+const InfoTerm = styled.dt(({ theme }) => ({
+  ...theme.typography.subtitle2,
+  textWrap: "nowrap",
+}))
+
+const InfoDetail = styled.dd(({ theme }) => ({
+  ...theme.typography.body2,
+  margin: "0 0 16px 0",
+}))
+
+const Verify = styled.div(({ theme }) => ({
+  display: "flex",
+  justifyContent: "center",
+  borderTop: `1px solid ${theme.custom.colors.lightGray2}`,
+  paddingTop: "16px",
+}))
+
+const StyledButtonLink = styled(ButtonLink)(({ theme }) => ({
+  backgroundColor: theme.custom.colors.white,
+}))
+
+/*
+ * This interface defines the structure of a VerifiableCredential (Open Badges v3.0).
+ * While this type is not explicitly defined / unknown in our API, we can rely on its stability
+ * because the structure and fields are mandated by the Open Badges v3.0 specification.
+ *
+ * References:
+ * - https://www.imsglobal.org/spec/ob/v3p0#verifiablecredential
+ * - https://www.imsglobal.org/spec/ob/v3p0#complete-openbadgecredential
+ * - https://github.com/digitalcredentials/mit-learn-obv3-template/tree/main?tab=readme-ov-file#recommended-properties-for-obv3
+ *
+ * We may type it on the API in a later revision.
+ */
+export interface VerifiableCredential {
+  id: string
+  type: string[]
+  proof?: {
+    type: string
+    created: string
+    proofValue: string
+    cryptosuite: string
+    proofPurpose: string
+    verificationMethod: string
+  }
+  issuer: {
+    id: string
+    name: string
+    type: string[]
+    image?: {
+      id: string
+      type: string
+      caption?: string
+    }
+  }
+  "@context": string[]
+  validFrom: string
+  validUntil: string
+  credentialSubject: {
+    type: string[]
+    identifier: Array<{
+      salt: string
+      type: string
+      hashed: boolean
+      identityHash: string
+      identityType: string
+    }>
+    achievement: {
+      id: string
+      name: string
+      type: string[]
+      image?: {
+        id: string
+        type: string
+        caption?: string
+      }
+      criteria?: {
+        narrative: string
+      }
+      description: string
+      achievementType: string
+    }
+    activityEndDate?: string
+    activityStartDate?: string
+  }
+}
+
+export const DigitalCredentialDialog = ({
+  verifiableCredential,
+  open,
+  onClose,
+}: {
+  verifiableCredential: VerifiableCredential
+  open: boolean
+  onClose: () => void
+}) => {
+  const { issuer, validFrom, validUntil, credentialSubject } =
+    verifiableCredential
+  const { identifier, achievement } = credentialSubject
+  return (
+    <Dialog
```

Comment:
The Verify Credential and Cancel / Download Digital Credential buttons clash here if the window is too small:

<img width="1278" height="630" alt="image" src="https://github.com/user-attachments/assets/2b401604-f956-4d39-a032-e037cb67bdb7" />

I know the dialog uses absolute positioning to set where the buttons are, but we should figure out a way to avoid this happening.